src/interface/interface.tsx

```diff
@@ -0,0 +1,25 @@
+//todo 인터페이스 정의
+export interface ITodo {
+  id: number;
+  text: string;
+  isCompleted: boolean;
+}
+
+//handleTodoInput 인터페이스 정의
+export interface ITodoInputTypes {
+  handleTodoInput(todoText: string): void;
+}
+
+// todo, handleTodoDelete, handleTodoToggle 인터페이스 정의
+export interface ITodoItemTypes {
+  todo: ITodo;
+  handleTodoDelete(id: number): void;
+  handleTodoToggle(id: number): void;
+}
+
+// todolist, handleTodoDelete, handleTodoToggle 인터페이스 정의
+export interface ITodoListsTypes {
```

Comment:
타입스크립트 기초에 대해 공부하면서 알게되었는데 export를 한번에 해주는게 일반적이라고 합니다!

```suggestion
export type { 
  ITodo,
  ITodoInputTypes,
  ITodoItemTypes,
  ITodoListsTypes,
}
```

Reply:
와.. 이렇게 하면 하나하나 export를 작성하지 않아도 되는군요...!!! 공부가 부족했던 부분이네요!! 감사합니다!!!!